_tickers.md

```diff
@@ -11,70 +11,424 @@ curl https://api.uphold.com/v0/ticker/USD
 
 ```json
 [{
-  "ask": "225.92",
-  "bid": "225.8",
+  "ask": "0.27224",
+  "bid": "0.27224",
+  "currency": "USD",
+  "pair": "AEDUSD"
+},
+{
+  "ask": "0.0246",
+  "bid": "0.0246",
+  "currency": "USD",
+  "pair": "ARSUSD"
+},
+{
+  "ask": "0.72595",
+  "bid": "0.72595",
+  "currency": "USD",
+  "pair": "AUDUSD"
+},
+{
+  "ask": "0.21258",
+  "bid": "0.21151",
+  "currency": "USD",
+  "pair": "BATUSD"
+},
+{
+  "ask": "532.82",
+  "bid": "530.96",
+  "currency": "USD",
+  "pair": "BCHUSD"
+},
+{
+  "ask": "0.23772",
+  "bid": "0.23772",
+  "currency": "USD",
+  "pair": "BRLUSD"
+},
+{
+  "ask": "6861.79",
+  "bid": "6860.42",
   "currency": "USD",
   "pair": "BTCUSD"
-}, {
-  "ask": "1.0693",
-  "bid": "1.0692",
+},
+{
+  "ask": "21.4774",
+  "bid": "21.37089",
+  "currency": "USD",
+  "pair": "BTGUSD"
+},
+{
+  "ask": "0.7697",
+  "bid": "0.7697",
+  "currency": "USD",
+  "pair": "CADUSD"
+},
+{
+  "ask": "1.0301",
+  "bid": "1.0301",
   "currency": "USD",
   "pair": "CHFUSD"
-}, {
-  "ask": "1.1266",
-  "bid": "1.1258",
+},
+{
+  "ask": "0.14609",
+  "bid": "0.14609",
+  "currency": "USD",
+  "pair": "CNYUSD"
+},
+{
+  "ask": "183.89597",
+  "bid": "183.17321",
+  "currency": "USD",
+  "pair": "DASHUSD"
+},
+{
+  "ask": "0.15621",
+  "bid": "0.15621",
+  "currency": "USD",
+  "pair": "DKKUSD"
+},
+{
+  "ask": "278.82",
+  "bid": "278.46",
+  "currency": "USD",
+  "pair": "ETHUSD"
+},
+{
+  "ask": "1.16485",
+  "bid": "1.16485",
   "currency": "USD",
   "pair": "EURUSD"
-}, {
-  "ask": "1.5329",
-  "bid": "1.5328",
+},
+{
+  "ask": "1.29944",
+  "bid": "1.29944",
   "currency": "USD",
   "pair": "GBPUSD"
-}, {
-  "ask": "0.0646",
-  "bid": "0.0646",
+},
+{
+  "ask": "0.1274",
+  "bid": "0.1274",
+  "currency": "USD",
+  "pair": "HKDUSD"
+},
+{
+  "ask": "0.27672",
+  "bid": "0.27672",
+  "currency": "USD",
+  "pair": "ILSUSD"
+},
+{
+  "ask": "0.01407",
+  "bid": "0.01407",
+  "currency": "USD",
+  "pair": "INRUSD"
+},
+{
+  "ask": "0.00899",
+  "bid": "0.00899",
+  "currency": "USD",
+  "pair": "JPYUSD"
+},
+{
+  "ask": "0.00994",
+  "bid": "0.00994",
+  "currency": "USD",
+  "pair": "KESUSD"
+},
```

Comment:
The LBAUSD pair is missing here.

Reply:
Makes sense, it's a token that has been added after this PR. Will add 👍